From: Suren Baghdasaryan <surenb@google.com>
Subject: mm/mempool: Documentation: add missing mempool_create_node documentation
Date: Tue, 2 Apr 2024 11:08:35 -0700

mempool_create_node documentation for two of its parameters is missing.
Document those parameters.

Link: https://lkml.kernel.org/r/20240402180835.1661905-1-surenb@google.com
Signed-off-by: Suren Baghdasaryan <surenb@google.com>
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202404010132.6v0zt6oa-lkp@intel.com/
Tested-by: Kees Cook <keescook@chromium.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/mempool.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/mm/mempool.c~mempool-hook-up-to-memory-allocation-profiling-fix-2
+++ a/mm/mempool.c
@@ -256,6 +256,8 @@ EXPORT_SYMBOL(mempool_init_noprof);
  * @alloc_fn:  user-defined element-allocation function.
  * @free_fn:   user-defined element-freeing function.
  * @pool_data: optional private data available to the user-defined functions.
+ * @gfp_mask:  memory allocation flags
+ * @node_id:   numa node to allocate on
  *
  * this function creates and allocates a guaranteed size, preallocated
  * memory pool. The pool can be used from the mempool_alloc() and mempool_free()
_
